rdf2vec/_rdf2vec.py

```diff
@@ -72,21 +56,21 @@ def __init__(self, vector_size=500, walkers=RandomWalker(2, float('inf')),
         self.min_count = min_count
 
     def fit(self, graph, instances):
-        """Fit the embedding network based on provided instances.
-        
-        Parameters
-        ----------
-        graphs: graph.KnowledgeGraph
-            The graph from which we will extract neighborhoods for the
-            provided instances. You can create a `graph.KnowledgeGraph` object
-            from an `rdflib.Graph` object by using a converter method.
-
-        instances: array-like
-            The instances for which an embedding will be created. It important
-            to note that the test instances should be passed to the fit method
-            as well. Due to RDF2Vec being unsupervised, there is no 
-            label leakage.
-        -------
+        """Fits the embedding network based on provided instances.
+
+        Note:
```

Comment:
Perhaps this note should be included closer around the parameter `graph` as it is related to that?